docs/guides/code/max_tasks_per_minute.py

```diff
@@ -0,0 +1,14 @@
+from crawlee import ConcurrencySettings
+from crawlee.beautifulsoup_crawler import BeautifulSoupCrawler
+
+concurrency_settings = ConcurrencySettings(
+    # Let the crawler know it can run up to 100 requests concurrently at any time
+    max_concurrency=100,
+    # ...but also ensure the crawler never exceeds 250 requests per minute
+    max_tasks_per_minute=10,
+)
+
+crawler = BeautifulSoupCrawler(
+    # Pass the concurrency setting to the crawler
+    concurrency_settings=concurrency_settings,
+)
```

Comment:
```suggestion
import asyncio

from crawlee import ConcurrencySettings
from crawlee.beautifulsoup_crawler import BeautifulSoupCrawler


async def main() -> None:
    concurrency_settings = ConcurrencySettings(
        # Set the maximum number of concurrent requests the crawler can run to 100.
        max_concurrency=100,
        # Limit the total number of requests to 10 per minute to avoid overwhelming
        # the target website.
        max_tasks_per_minute=10,
    )

    crawler = BeautifulSoupCrawler(
        # Apply the defined concurrency settings to the crawler.
        concurrency_settings=concurrency_settings,
    )


if __name__ == '__main__':
    asyncio.run(main())
```